github_copilot_seats/triggers.py

```diff
@@ -0,0 +1,32 @@
+from autokitteh.slack import slack_client
+from seats import prune_idle_seats, find_idle_seats
+
+s = slack_client("slack_conn")
+
+
+def on_schedule() -> None:
+    "Entry point for the workflow"
+    print(prune_idle_seats())
+
+
+def on_slack_slash_command(event) -> None:
+    "Entry point for the workflow"
+    cmd = event.data.text
+    cid = event.data.channel_id
+
+    if cmd not in {"prune-idle-copilot-seats", "find-idle-copilot-seats"}:
```

Comment:
never seen `{}` used in this way, it think it means a set? no need to be fancy here. i think `[]` or `{}` is more customary.